src/home.tsx

```diff
@@ -0,0 +1,23 @@
+// let maxWidth = 0;
```

Comment:
Any reason for still keeping this file then?

Reply:
Yeah we can remove the file but in the future we may use it. All components are now written in HTML but it might be better to rewrite it to React in the future

Reply:
Ah, I see, well then let's keep it for now 👍🏻